packages/calypso-products/src/plans-list.js

```diff
@@ -1369,6 +1375,30 @@ export const PLANS_LIST = {
 		getProductId: () => 2015,
 	},
 
+	[ PLAN_JETPACK_SECURITY ]: {
+		group: GROUP_JETPACK,
+		type: TYPE_SECURITY,
+		...getAnnualTimeframe(),
+	},
+
+	[ PLAN_JETPACK_SECURITY_MONTHLY ]: {
+		group: GROUP_JETPACK,
+		type: TYPE_SECURITY,
+		...getMonthlyTimeframe(),
+	},
+
+	[ PLAN_JETPACK_SECURITY_PRO ]: {
+		group: GROUP_JETPACK,
+		type: TYPE_SECURITY_PRO,
+		...getAnnualTimeframe(),
+	},
+
+	[ PLAN_JETPACK_SECURITY_PRO_MONTHLY ]: {
+		group: GROUP_JETPACK,
+		type: TYPE_SECURITY_PRO,
+		...getMonthlyTimeframe(),
+	},
```

Comment:
You probably know this, so feel free to disregard this comment, but properties are missing here. I guess it's because we don't know for sure what features each new product will have. If not having them creates trouble, you could use `getPlanJetpackSecurityRealtimeDetails` to populate them.

Reply:
Yeah I was leaving those out for now, since a lot of them are translated strings and I don't want commit translated strings that aren't finalized yet.